src/api/app.py

```diff
@@ -461,6 +468,22 @@ def update_thread(
         return ThreadModel.model_validate(thread)
 
 
+@app.post("/api/custom_area_name")
+async def custom_area_name(
+    request: CustomAreaNameRequest, user: UserModel = Depends(fetch_user)
+):
+    """
+    Generate a neutral geographic name for a GeoJSON FeatureCollection of bounding boxes.
+    Requires Authorization.
+    """
+    try:
+        prompt = f"Name this GeoJSON FeatureCollection neutrally by geography, make sure to check where they are:\n{request.model_dump()}\n return strictly name only."
```

Comment:
Should we tell it to be concise / limit itself to 100 / 150 chars or so? Am worried this might sometimes generate overly verbose names?

Reply:
Yeah, that's a good idea. I can edit